src/xSTIR/cSTIR/tests/test1.cpp

```diff
@@ -125,6 +127,11 @@ int test1()
 		am.set_background_term(sptr_b);
 		am.set_up(sptr_ad, sptr_id);
 
+		// compute the norm of the linear part of the acquisition model
+		std::cout << "computing the norm of the linear part of the acquisition model...\n";
+		float lmd = am.norm();
+		std::cout << "acquisition model operator norm: " << lmd << '\n';
+
```

Comment:
can we add a test on the actual norm? Presumably something like
```
y = Ax
norm(y)<=norm(A)*norm(x)
```
for some random x.

Reply:
done